core/insertion_marker_manager.ts

```diff
@@ -51,87 +60,79 @@ const DUPLICATE_BLOCK_ERROR = 'The insertion marker ' +
  * @alias Blockly.InsertionMarkerManager
  */
 export class InsertionMarkerManager {
-  private readonly topBlock_: BlockSvg;
-  private readonly workspace_: WorkspaceSvg;
+  /**
+   * The top block in the stack being dragged.
+   * Does not change during a drag.
+   */
+  private readonly topBlock: BlockSvg;
+
+  /**
+   * The workspace on which these connections are being dragged.
+   * Does not change during a drag.
+   */
+  private readonly workspace: WorkspaceSvg;
 
   /**
    * The last connection on the stack, if it's not the last connection on the
    * first block.
    * Set in initAvailableConnections, if at all.
    */
-  private lastOnStack_: RenderedConnection|null = null;
+  private lastOnStack: RenderedConnection|null = null;
 
   /**
    * The insertion marker corresponding to the last block in the stack, if
    * that's not the same as the first block in the stack.
    * Set in initAvailableConnections, if at all
    */
-  private lastMarker_: BlockSvg|null = null;
-  private firstMarker_: BlockSvg;
+  private lastMarker: BlockSvg|null = null;
 
   /**
-   * The connection that this block would connect to if released immediately.
-   * Updated on every mouse move.
-   * This is not on any of the blocks that are being dragged.
+   * The insertion marker that shows up between blocks to show where a block
+   * would go if dropped immediately.
    */
-  private closestConnection_: RenderedConnection|null = null;
+  private firstMarker: BlockSvg;
 
   /**
-   * The connection that would connect to this.closestConnection_ if this
-   * block were released immediately. Updated on every mouse move. This is on
-   * the top block that is being dragged or the last block in the dragging
-   * stack.
+   * Information about the connection that would be made if the dragging block
+   * were released immediately. Updated on every mouse move.
    */
-  private localConnection_: RenderedConnection|null = null;
+  private activeCandidate: CandidateConnection|null = null;
 
   /**
    * Whether the block would be deleted if it were dropped immediately.
    * Updated on every mouse move.
    */
-  private wouldDeleteBlock_ = false;
+  private wouldDeleteBlockInternal = false;
```

Comment:
I don't like this property name, especially the `Internal` bit which suggests it is an internal implementation/alternative to the `wouldDeleteBlock` method, but it turns out not to be a method but a boolean.

Does the `wouldDeleteBlock` method, being marked `@internal` serve any useful purpose?  Could it be removed, this property renamed `wouldDeleteBlock`, and calls to the former method replaced with property accesses?  (There might even be a marginal performance benefit in avoiding method calls in what I am guessing is performance-sensitive code.)

If changes to `public @internal` properties are considered to be breaking (because someone might have ignored the `@internal` tag), then consider rename this to `deleteBlockOnDrop` or the like.

Reply:
I went with `wouldDeleteOnDrop`.

re: property access vs performance benefit: it's essentially acting as a getter right now, which is nice because alternate implementations could change the implementation of `wouldDeleteBlock`. But when I say it out loud, that sounds a lot like designing for a problem we haven't hit. 

@maribethb my position on `public @internal` is that it's not breaking to change them, because during migration we removed `@internal` from things that were actually `public`, and anything left is actually `@internal`. Do you agree?

Reply:
Agreed, things marked `@internal` should not be considered public and therefore you can change them without it being breaking.

I agree with Christopher's suggestion to remove the get method and just use a property. Re: alternate implementation, is there actually a way to have an alternate implementation (e.g. can you register a custom insertion marker manager?)? if not, then that is more reason why it's not a problem we need to solve, and if so then we actually need to be careful because this might be `protected` rather than `internal`?

Reply:
Done.